daffodil-io/src/main/scala/org/apache/daffodil/io/InputSourceDataInputStream.scala

```diff
@@ -614,40 +614,23 @@ final class InputSourceDataInputStream private(val inputSource: InputSource)
     markPool.finalCheck
   }
 
-  final def getString(nChars: Long, finfo: FormatInfo): Maybe[String] = {
-    val startingBitPos = bitPos0b
-    val aligned = align(finfo.encodingMandatoryAlignmentInBits, finfo)
-    if (!aligned) {
-      Maybe.Nope
-    } else {
-      withLocalCharBuffer { lcb =>
-        val cb = lcb.getBuf(nChars)
-        val numDecoded = finfo.decoder.decode(this, finfo, cb)
-        if (numDecoded == nChars) {
-          Maybe(cb.flip.toString)
-        } else {
-          setBitPos0b(startingBitPos)
-          Maybe.Nope
-        }
-      }
-    }
-  }
-
+  /**
+   * Almost the same as getString. But returns some characters (up to nChars) if they are available.
```

Comment:
I believe you've removed getString so this comment should just say "Returns some...", yes?